app/utils/deep_link/index.ts

```diff
@@ -115,7 +115,11 @@ export async function handleDeepLink(deepLinkUrl: string, intlShape?: IntlShape,
 }
 
 export function parseDeepLink(deepLinkUrl: string): DeepLinkWithData {
-    const url = removeProtocol(deepLinkUrl);
+    const url = removeProtocol(decodeURIComponent(deepLinkUrl));
```

Comment:
Don't we need to catch the exception where URI is malformed: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent#exceptions ?

Reply:
thank you for spotting this @esarafianou